ncs/rad_envelope.yaml.jinja2

```diff
@@ -1,5 +1,10 @@
-{%- set mpi_radio_vendor_name = application['config']['CONFIG_SUIT_MPI_RAD_LOCAL_1_VENDOR_NAME']|default('nordicsemi.com') %}
-{%- set mpi_radio_class_name = application['config']['CONFIG_SUIT_MPI_RAD_LOCAL_1_CLASS_NAME']|default('nRF54H20_sample_rad') %}
+{%- if application is not defined %}
+  {%- set main_config = radio %}
+{%- else %}
+  {%- set main_config = application %}
+{%- endif %}
+{%- set mpi_radio_vendor_name = main_config['config']['CONFIG_SUIT_MPI_RAD_LOCAL_1_VENDOR_NAME']|default('nordicsemi.com') %}
```

Comment:
If the main purpose is to use this config for MPIs, you may consider changing `main_config` to `mpi_config` :slightly_smiling_face:

Reply:
Let's stay with main_config :)